modules/client-configuration/pages/contact-methods-saltssh.adoc

```diff
@@ -4,16 +4,26 @@
 
 Push via Salt SSH is used in environments where Salt clients cannot reach the {productname} Server directly.
 In this environment, clients are located in a firewall-protected zone called a DMZ.
-No system within the DMZ is authorized to open a connection to the internal network, including the {productname} Server.
+No system within the DMZ is authorized to open a connection to the internal network  where the {productname} Server is located.
 
-The Push via Salt SSH method creates an encrypted tunnel from the {productname} Server on the internal network to the clients located on the DMZ.
-After all actions and events are executed, the tunnel is closed.
+Push via Salt SSH is also to use if installing a daemon agent on clients is not possible.
+
+[WARNING]
+====
+The Push via Salt SSH method has serious limitations.
+It does not scale well, and consumes more Server resources and network bandwidth than the plain Salt method.
+The Push via Salt SSH method is not at all supported with large setups (1000 clients and more).
+====
+
+The Push via Salt SSH method creates an encrypted tunnel from the {productname} Server on the internal network to the clients located in the DMZ.
+After all actions such as updates and events are pushed and executed, the tunnel is closed.
```

Comment:
We have "SSH Push" and "SSH Push with Tunnel". Not sure if we say this somewhere.
"SSH Push" opens only a tunnel from SUMA to Client, but the reverse channel goes directly. This might not work everywhere. For this we have "SSH Push with Tunnel". This opens also a reverse tunnel and you can break through all firewalls which might block the connection from client to server.

Reply:
Ok, thanks.  Yes, we must add this info.  I created a separate issue, see below.